packages/terra-form-fieldset/src/Fieldset.scss

```diff
@@ -0,0 +1,41 @@
+:local {
+  .fieldset {
+    border: 0;
+    margin-bottom: 1.428em;
+    padding: 0;
+  }
+
+  .fieldset-required > .legend:first-child {
+    &::before {
+      color: var(--terra-form-fieldset-required-color, #e50000);
+      content: '*';
+      display: inline-block;
+      padding-right: 0.357em;
+    }
+  }
+
+  .legend {
+    border: 0;
+    color: var(--terra-form-fieldset-legend-color);
+    display: block;
+    font-weight: var(--terra-form-fieldset-legend-font-weight, bold);
+    margin-bottom: 0.357em;
```

Comment:
should this be themeable?

Reply:
Discussed with @bjankord about theming margins and  we decided not to theme margins untill stated explicitly in the callouts (which we don't have now).